### ISSUE TITLE
Silence "Opening for reading"

### ISSUE THREAD

**Issue by davispuh** [ytdl-org/youtube-dl#25165]
## Checklist

- [ x] I'm reporting a feature request
- [ x] I've verified that I'm running youtube-dl version **2020.05.03**
- [ x] I've searched the bugtracker for similar feature requests including closed ones

## Description

Currently when downloading HLS stream from `.m3u8` file you get all console spammed with loads of 
```
[https @ 0x557f165b4b40] Opening 'https://server/file.mp4/segmentXXXX_0_a.ts' for reading  
```

I want to silent them so that I can still see download progress. If I use `-q` then there's no progress information anymore.

So basically what I want is `-q --progress` so it's quiet but with progress information.

**Comment by cookieguru** [comment]
If I'm not mistaken that message actually comes from ffmpeg.  You could pass `-loglevel warning` to ffmpeg via `--external-downloader-args`

**Comment by zackmark29** [comment]
> If I'm not mistaken that message actually comes from ffmpeg. You could pass `-loglevel warning` to ffmpeg via `--external-downloader-args`

What is the parameter for that? only --external-downloader-args?

**Comment by cookieguru** [comment]
There are examples of its usage in other issues here.  GitHub's built-in search will find them.

**Comment by davispuh** [comment]
Awesome, so turns out everything is done by `ffmpeg`.
Using just `-loglevel warning` didn't work because it also hide progress, but this works perfectly and is exactly what I needed:
`youtube-dl --external-downloader-args "-loglevel warning -stats"`

I guess it would be nice if youtube-dl had flag for this case (quiet but with progress) but it's not really issue as can achieve it anyway.
